lib/velum/secrets.rb

```diff
@@ -0,0 +1,16 @@
+require "pathname"
+require "securerandom"
+
+module Velum
+  # If there is a rails key base stored on disk, return the stored one.
+  # Otherwise, create a new key base from cryptographic random pool and store it.
+  class Secrets
+    def self.read_or_create_secret_key_base(key_path)
+      key_path = Pathname.new(key_path)
+      key_path.dirname.mkpath
+      return key_path.read if key_path.exist?
+      key_path.write(SecureRandom.hex(64))
+      key_path.read
```

Comment:
We could do here:

```
SecureRandom.hex(64).tap do |secret_key_base|
  key_path.write secret_key_base
end
```
Instead of `key_path.write` and `key_path.read`, no?